_data/en/settings.yml

```diff
@@ -261,6 +261,8 @@ nav:
         url: /policy/our-security-practices/
       - name: Messaging terms and conditions
         url: /policy/messaging-terms-and-conditions/
+      - name: Rules of use
```

Comment:
We need this in `_data/fr/settings.yml` and `_data/es/settings.yml` as well, so that the page shows up as a sidebar link in those languages.

Reply:
great catch!